utests/__init__.py

```diff
@@ -1 +1,8 @@
-from exarl.agents.agent_vault.dqn import DQN
+from exarl.utils import candleDriver
+try:
+    
+    # candleDriver.initialize_parameters()
+    print("Loaded this here")
```

Comment:
Are we missing something here?

Reply:
Also on it.

Reply:
Cleaned this up, but this file is empty now on purpose.